circle.yml

```diff
@@ -19,14 +19,14 @@ deployment:
   staging:
     branch: develop
     commands:
-      - git push git@heroku.com:code-corps-phoenix-staging.git $CIRCLE_SHA1:master
-      - heroku run "POOL_SIZE=2 mix ecto.migrate" --app code-corps-phoenix-staging
+      - git push git@heroku.com:code-corps-staging.git $CIRCLE_SHA1:master
+      - heroku run "POOL_SIZE=2 mix ecto.migrate" --app code-corps-staging
```

Comment:
I do not know how our heroku stuff is set up. I just want to confirm this is supposed to be `code-corps-staging` and not something like `code-corps-api-staging`